install/updates/froxlor/0.9/update_0.9.inc.php

```diff
@@ -3002,5 +3002,7 @@
     showUpdateStep("Updating from 0.9.34-dev4 to 0.9.34 final");
     lastStepStatus(0);
 
+	Database::query("ALTER TABLE `".TABLE_PANEL_DOMAINS."` ADD `dnsrecords` text;");
```

Comment:
Wrong place for the database update, people using 0.9.34 final or later will NEVER get this update, always append a new -devX version when db updates are needed (e.g 0.9.35-dev1)